# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/apz/test/mochitest/browser_test_group_fission.js
# Commit: 2e296c018c3b
# Full Hash: 2e296c018c3b62984150b42cec757af680d18695
# Author: Hiroyuki Ikezoe <hikezoe.birchill@mozilla.com>
# Date: 2021-02-02 03:35:00
# Regressor Bug: 1687067
# File Overlap Count: 1
# Description:
#   Bug 1687067 - Build OverscrollHandoffChain by walking up hit testing tree instead of APZC tree. r=botond
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D103601
# ==============================================================================

diff -r a32d58612fb7 -r 2e296c018c3b gfx/layers/apz/test/mochitest/browser_test_group_fission.js
--- a/gfx/layers/apz/test/mochitest/browser_test_group_fission.js	Mon Feb 01 23:23:42 2021 +0000
+++ b/gfx/layers/apz/test/mochitest/browser_test_group_fission.js	Mon Feb 01 23:25:17 2021 +0000
@@ -59,6 +59,7 @@
       file: "helper_fission_tap_in_nested_iframe_on_zoomed.html",
       prefs: [["apz.max_tap_time", 10000]],
     },
+    { file: "helper_fission_scroll_handoff.html" },
     // add additional tests here
   ];
   if (isWebRender) {